custom/icds_reports/views.py

```diff
@@ -554,22 +554,23 @@ def get(self, request, *args, **kwargs):
         if age:
             config.update(get_age_filter(age))
 
-        config.update(get_location_filter(location, domain))
+        location_dict = get_location_filter(location, self.kwargs['domain'])
```

Comment:
Why does this need to be a separate variable. The config, with all this info, is already passed to all these functions. You can just past that on to and pull out the fields you need in `get_location_launched_status`. I think that would cut the size of this change down dramatically.

Reply:
Um. I could do that. But I think that wont be very straightforward, Because, location filters given by `get_location_filter` are multiple level filters like if agg_level=3 then locations filters will be state_id=xyz,district_id=xyz,block_id=xyz. And I need all 4 of them in  `get_location_launched_status`. and pulling these from `config` would not be very short as it contains other fields as well specific to particular reports

Reply:
ow, if you mean the diff will decrease. I agree. Cool. Let me make that change.

Reply:
done with the change